lib/index.ts

```diff
@@ -0,0 +1,39 @@
+import { clientInfo } from './environment'
+
+declare var PercyAgent: any;
+declare var Nightmare: any;
+
+/**
+ * A function to take a Percy snapshot from a NightmareJS test. To use in your tests:
+ *   const { percySnapshot } = require('@percy/nightmare')
+ *   [...]
+ *   nightmare
+ *     .goto(<yourtesturl>)
+ *     .use(percySnapshot(<your snapshot name>), <maybe options>))
+ *
+ * @param name Name of the snapshot that we're taking.
+ * @param options Additional options, e.g. '{widths: [300, 600, 1000]}'. Optional.
+ */
+export function percySnapshot(name: string, options: any = {}) {
+  const percyClientInfo: string =  clientInfo()
+  return function (nightmare: any) {
+    nightmare
+      .inject('js', 'node_modules/@percy/nightmare/node_modules/@percy/agent/dist/public/percy-agent.js')
+      .evaluate(function (name: string, options: any, percyClientInfo: string) {
+        name = name || document.title
+        var nightmareAgent = function() {
+          try {
+              return window.clientInformation.userAgent
```

Comment:
This generates the standard long agent string, so this ends up looking like:
```
"environmentInfo":"nightmare/Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Electron/1.8.8 Safari/537.36
```
I don't love it, and I'm not sure it's useful. I would suggest we just leave this as the constant `nightmare`, with no version. Happy to hear other ideas as well, @djones .

Reply:
Since `clientInfo` shows up like this `@percy/nightmare/0.0.1`, which gives us lots of information, I think we should just leave `environmentInfo` as empty.

Reply:
Done.